Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG PHP_VERSION
+ARG PHP_EXTENSIONS
+
+FROM php:${PHP_VERSION}-fpm
+
+WORKDIR /usr/local/etc/php/conf.d/
+COPY --link symfony.ini .
+WORKDIR /usr/local/etc/php/pool.d/
+COPY --link symfony.pool.conf .
+
+ADD --chmod=0755 https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏You should the script from the official docker image instead of downloading the latest 

```suggestion
COPY --from=extension-installer /usr/bin/install-php-extensions /usr/local/bin
```
And add your dependencies at the top : 

```diff
+ FROM mlocati/php-extension-installer:2 as extension-installer
  FROM php:${PHP_VERSION}-fpm as base
```
It is safer and you will have to update manually this dependency whenever a new major version is published

Reply:
I chose this approach from [their documentation](https://github.com/mlocati/docker-php-extension-installer?tab=readme-ov-file#usage) because of [the warning about the Docker method](https://github.com/mlocati/docker-php-extension-installer?tab=readme-ov-file#copying-the-script-from-a-docker-image), but as it is built in a CI, it should always pull the latest version of the image, so it should be fine.